app/views/rulesets/index.html.erb

```diff
@@ -11,8 +11,14 @@
             </h1>
           </legend>
           <div class="govuk-radios" data-module="govuk-radios">
+            <% Linters::Rulesets::ACTIVE_RULESETS.each do |ruleset_name| %>
+              <div class="govuk-radios__item">
+                <%= text_field_tag("ruleset_name", ruleset_name, class: "govuk-radios__input", type: "radio", name: "ruleset_name", checked: true) %>
```

Comment:
[suggestion]

Is there a reason why `text_field_tag` is used here. Isn't [`radio_button_tag`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-radio_button_tag) more appropriate?

Reply:
The radio button tag doesn't work with the gov component, or at least I couldn't get it to work.

Reply:
OK - though that seems strange.

Leave it as it is.